app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java

```diff
@@ -50,6 +51,7 @@
     }
 
     private static final String TAG = "ConfigXml";
+    private static final int DEVICE_ID_APPENDIX_LENGTH = 7;
```

Comment:
private static final int FOLDER_ID_APPENDIX_LENGTH = 4;

Name change; Suggest to make it 4 chars long as this only applies for the default camera folder (one single folder created once on first launch) How much chance do we have 4 random char's collide? I think that's okay.

Reply:
Yes, probably it's ok to have an appendix 4 chars long. We have around ~1/10^6 chance that two same devices will be given the same id. Moreover, it's an instance of the generalized birthday problem and we need ~450 same devices to have at least one pair of devices with the same id with probability 1/10.